stub/src/main/java/io/grpc/stub/ClientCalls.java

```diff
@@ -211,6 +218,95 @@ public static <ReqT, RespT> Iterator<RespT> blockingServerStreamingCall(
     return result;
   }
 
+  /**
+   * Initiates a client streaming call over the specified channel.  It returns an
+   * object which can be used in a blocking manner to send values to the server and retrieve a
+   * response.
+   *
+   * <p>Call {@link BlockingClientCall#write} for each value to send to the server. After the last
+   * value has been written, call {@link BlockingClientCall#halfClose} to indicate that writing is
+   * complete and then {@link BlockingClientCall#read} to get the response.
+   *
+   * <p>The methods {@link BlockingClientCall#hasNext()} and {@link
+   * BlockingClientCall#cancel(String, Throwable)} can be used for more extensive control.
+   *
+   * @return A {@link BlockingClientCall} that has had the request sent and halfClose called
+   * @throws InterruptedException if it receives an interrupt while sending the request
+   * @throws StatusException if the write to the server failed
+   */
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/10918")
+  public static <ReqT, RespT> BlockingClientCall<?, RespT> blockingV2ServerStreamingCall(
```

Comment:
To allow us to change the `?` to `ReqT` in the generated code if we find it works better, we may want the `ReqT` present here but use `?` in the generated code. We can talk about that.